src/gsi/read_seviri.f90

```diff
@@ -103,7 +103,7 @@ subroutine read_seviri(mype,val_sev,ithin,rmesh,jsatid,&
   integer(i_kind) nchanl,ilath,ilonh,ilzah,iszah,irec,next
   integer(i_kind) nmind,lnbufr,idate,ilat,ilon,nhdr,nchn,ncld,nbrst,jj
   integer(i_kind) ireadmg,ireadsb,iret,nreal,nele,itt
-  integer(i_kind) itx,i,k,isflg,kidsat,n,iscan,idomsfc
+  integer(i_kind) itx,i,k,isflg,kidsat,n,iscan,idomsfc,number_profiles
```

Comment:
`number_profiles` should be removed from `src/gsi/read_seviri.f90`, right?

Reply:
@RussTreadon-NOAA Correct.  I am in the process of removing it.  I will look for others.

Reply:
All the number_profiles should be removed.  I pushed the commit up to github.
Will close this conversation.